### ISSUE TITLE
Files that are outside of the Source folder are copied into Build folder

### ISSUE THREAD

**Issue by heartfield** [bdkjones/CodeKit#343]
**Quick, short summary:**

When using the Source/Build folder process, files and folders outside of the Source folder (but in  root level of project folder) are processed and copied into the Build folder.

**Expected results:**
I expected Codekit to **only** process files that are in the Source folder and ignore any other files/folder that are in the root of the project folder.

**Actual results:**
Any file or folder in the root folder of the project are processed/copied into the Build folder when using the Build process.

**Exact steps to reproduce:**
Using the Source/Build Process feature with default settings.

**Comment by luxlogica** [comment]
*Every* single developer I've introduced to CodeKit 3 has the expectation that only what's in 'source' will be processed, and are surprised to find out that files (and folders) outside 'source' get processed, too.

**Comment by dwkns** [comment]
Agreed. Maybe this could be the default behaviour when using build folders?

**Comment by fnstirling** [comment]
Agreed. Also curious if this feature request is going to be looked at?

**Comment by bdkjones** [comment]
Yep. I plan to introduce an option for it. Hopefully in the next release.

**Comment by fnstirling** [comment]
Great to hear

**Comment by oppenheimer-** [comment]
This would be cool indeed

**Comment by UniquePixels** [comment]
Was this implemented, seems to still be an issue as far as I can tell.
